include/deal.II/grid/manifold_lib.h

```diff
@@ -503,6 +503,60 @@ class TorusManifold : public ChartManifold<dim,3,3>
   double r, R;
 };
 
+
+
+/**
+ * A mapping class that smoothly transitions from different manifold
+ * descriptions on different faces by a so-called transfinite
+ * interpolation. As opposed to setting the manifold description only on the
+ * boundary of a mesh, this approach makes sure that the deformation of the
+ * mesh according to a curved boundary description is applied to more than one
+ * element layer once the mesh gets refined, which is necessary to get
```

Comment:
this is more than one *fine* mesh layer, right? But only one *coarse* mesh layer.